scripts/update-chart-version.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -xeuo pipefail
+
+# Use the travis variables when available because travis clones different than what is on a local dev machine
+# VERSION = the tag if present, otherwise the short commit hash
+# BRANCH = the current branch, empty if not on a branch
+if [[ "${TRAVIS-}" == "true" ]]; then
```

Comment:
@damoon we are not using Travis-CI anymore for our CI. Instead, we're using CircleCI. According to their [built-in environment variables](https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables) documentation, I think this would be `CI` instead of `TRAVIS`.

Similarly, instead of the below `TRAVIS_TAG`, you could use `CIRCLE_TAG`

What do you think?